### DISCUSSION TITLE
Process for removing experimental status

### DISCUSSION THREAD

**Discussion by pendo324** [lima-vm/lima#1601] [General]
Do we have any process for removing the experimental status from a feature? Is it a case by case basis? I'm interested in contributing to promote some features regarding Virtualization.framework to "stable" status if we can get a list of work items. Maybe we should have a tracking issue for what experimental means per feature?

**Comment by jandubois** [comment]
So far it is an ad-hoc and case-by-case process. I believe the criteria for features graduating are:

* Does the feature work reliably?
* Is there a risk of data-loss?
* Does the feature have limitations that we don't know yet how to address?
* Are we sure the commandline API and the `lima.yaml` schema changes are final and won't have to change in an incompatible way?
* Are we waiting on upstream changes that might affect how we can expose the feature?

Anything else?

I agree that having a list of all experimental features, and the reasons why they are still considered experimental would be helpful.

**Comment by AkihiroSuda** [comment]
>  I'm interested in contributing to promote some features regarding Virtualization.framework to "stable" status if we can get a list of work items. Maybe we should have a tracking issue for what experimental means per feature?

Sure:
- https://github.com/lima-vm/lima/issues/1610
